cvat/apps/engine/signals.py

```diff
@@ -38,7 +38,10 @@ def __save_job_handler(instance, created, **kwargs):
 
 
 @receiver(post_save, sender=User, dispatch_uid=__name__ + ".save_user_handler")
-def __save_user_handler(instance: User, **kwargs):
+def __save_user_handler(instance: User, created: bool, raw: bool, **kwargs):
+    if created and raw:
```

Comment:
```suggestion
    if created or raw:
```

Reply:
Checked in a test run for [4a84adb](https://github.com/cvat-ai/cvat/pull/9272/commits/4a84adb04cc9890657d262637263826dfe902768)